fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/AtomicMutationIndexMaintainer.java

```diff
@@ -150,6 +155,8 @@ protected static AtomicMutation getAtomicMutation(@Nonnull Index index) {
                 }
             }
 
+            final byte[] compareAndClear = mutation.getCompareAndClearParam();
```

Comment:
minor: this feels like this local variable should be declared closer to where it's used (like right before it's checked for `null`-ity)

Reply:
Sure.